lib/packmap.js

```diff
@@ -9,6 +9,23 @@ module.exports = async function packmap(opts) {
   const packageJsonPath = path.resolve(process.cwd(), opts.package);
   const packageJson = require(packageJsonPath);
   const importMap = await traversePackage(packageJson);
+
+  if (opts.overrideMap) {
+    const overridingMapPath = path.resolve(process.cwd(), opts.overrideMap);
+    if (await fs.pathExists(overridingMapPath)) {
+      const overridingMap = require(overridingMapPath);
+      importMap.imports = Object.assign(
```

Comment:
This could be simplified to `Object.assign(importMap.imports, overridingMap.imports)`, or even just `Object.assign(importMap, overridingMap)`

Reply:
Oops! Changed to `This could be simplified to Object.assign(importMap.imports, overridingMap.imports)` as the later one will replace completely. Thanks